UI/BuildDesignatorWnd.cpp

```diff
@@ -1051,7 +1051,8 @@ void BuildDesignatorWnd::CenterOnBuild(int queue_idx, bool open) {
             MapWnd* map = ClientUI::GetClientUI()->GetMapWnd();
             map->CenterOnObject(system_id);
             if (open) {
-                SelectSystem(system_id);
+                HumanClientApp::GetApp()->GetClientUI().GetMapWnd()->SelectSystem(system_id);
+                //SelectSystem(system_id);
```

Comment:
The current source code is full of commented out lines, which are not maintained when refactoring and sometimes a reader can't even deduce the intent of the commented out code anymore because  the surrounding code evolved and the commit history is just convoluted or worse: Badly documented (hint @LGM-Doyle: Changed after PR Review #xxxx).

Consider now +2 years and you see this chunk of code:

> Yeah, probably the line was commented out because SelectSystem was broken, but fortunately I fixed this just a week ago.  Lets uncomment this.

Commented out code never adds any value, only confusion in the future.  Removing it in a different commit only adds additional ways to misinterpret the context and add even more confusion.